docs/azure/integrations-overview.md

```diff
@@ -104,6 +75,183 @@ The same is true for SQL and PostgreSQL services:
+
+When the app host executes, the [_execution context_](../fundamentals/app-host-overview.md#execution-context) determines whether the app host is in <xref:Aspire.Hosting.DistributedApplicationOperation.Run> or <xref:Aspire.Hosting.DistributedApplicationOperation.Publish> mode. The naming conventions for these APIs indicate the intended action for the resource. For more information on execution modes, see [Execution context](../fundamentals/app-host-overview.md#execution-context).
+
+The following table summarizes the naming conventions used to express Azure resources:
+
+| Operation | API | Description |
+|--|--|--|
+| Publish | `PublishAsConnectionString` | Changes the resource to be published as a connection string reference in the manifest. |
+| Publish | `PublishAsExisting` | Uses an existing Azure resource when the application is deployed instead of creating a new one. |
+| Run | `RunAsContainer` | Configures an equivalent container to run locally. For more information, see [Local containers](#local-containers). |
+| Run | `RunAsEmulator` | Configures the Azure resource to be emulated. For more information, see [Local emulators](#local-emulators). |
+| Run | `RunAsExisting` | Marks the resource as an existing resource when the application is running. |
+| Publish and Run | `AsExisting` | Marks the resource as an existing resource regardless of the operation. |
+
+> [!NOTE]
+> Not all APIs are available on all Azure resources. For example, some Azure resources can be containerized or emulated, while others can't.
+
+#### General run mode API use cases
+
+Use `RunAsExisting` when you need to dynamically interact with an existing resource during runtime without needing to deploy or update it. Use `PublishAsExisting` when declaring existing resources as part of a deployment configuration, ensuring the correct scopes and permissions are applied. Finally, use `AsExisting` when declaring existing resources in both configurations, with a requirement to parameterize the references.
+
+You can query whether a resource is marked as an existing resource, by calling the `IsExisting` extension method on the <xref:Aspire.Hosting.ApplicationModel.IResource>. For more information, see [Mark Azure resources as existing](#mark-azure-resources-as-existing).
+
+## Mark Azure resources as existing
+
+In addition to the preexisting <xref:Aspire.Hosting.ParameterResourceBuilderExtensions.AddConnectionString*> API, .NET Aspire provides expanded support for referencing existing Azure resources. You mark an existing resource through the `PublishAsExisting`, `RunAsExisting`, and `AsExisting` APIs. These APIs allow developers to reference already-deployed Azure resources, configure them, and generate appropriate deployment manifests using Bicep templates.
```

Comment:
```suggestion
.NET Aspire provides support for referencing existing Azure resources. You mark an existing resource through the `PublishAsExisting`, `RunAsExisting`, and `AsExisting` APIs. These APIs allow developers to reference already-deployed Azure resources, configure them, and generate appropriate deployment manifests using Bicep templates.
```

Since we haven't talked about Connection Strings yet, I wouldn't reference them here. In a couple months it won't matter what existed first.